src/init.lua

```diff
@@ -480,71 +484,63 @@ end
 	:::info
 ]=]
 
-function Store:LoadKeep(key: string, unReleasedHandler: UnReleasedHandler): Promise
+function Store:LoadKeep(key: string, unreleasedHandler: unreleasedHandler): Promise
 	local store = self._store
 
+	local validLoadMethods = {
+		ForceLoad = "ForceLoad",
+		-- maybe add "Ignore"?
+		Cancel = "Cancel",
+	}
```

Comment:
Maybe could globalize this or even a type if possible. Making it accessible may help users if docs aren't clear enough.